lab_2/main.py

```diff
@@ -43,7 +89,25 @@ def get_text_vector(original_text: list, language_profiles: dict) -> list or Non
     :param original_text: any tokenized text
     :param language_profiles: a dictionary of dictionaries - language profiles
     """
-    pass
+    if not isinstance(original_text, list):
+        return None
+    if not isinstance(language_profiles, dict):
+        return None
+    if language_profiles == {}:
+        return None
+
+    text_vector = []
+    unique_words = get_language_features(language_profiles)
+
+    for i, word in enumerate(unique_words):
+        if word in original_text:
+            for profile in language_profiles.values():
+                if word in profile.keys():
+                    text_vector.append(profile[word])
```

Comment:
there is a way to get a key if it exists or 0 if it doesn't

Reply:
For some reason it doesn't count in the last element. Please, look through what's wrong. I've been trying to find it for the past 30 mins.